### ISSUE TITLE
[BUG] Build with "Release6" branch fails

### ISSUE THREAD

**Issue by hojinp** [EQ-Alpha/KeyDB#381]
**Describe the bug**

When I try to build KeyDB with legacy FLASH, with `make MALLOC=memkind`, it makes the following error.
```
zmalloc.cpp:246:46: error: macro "realloc" requires 3 arguments, but only 2 given
     newptr = realloc(realptr,size+PREFIX_SIZE);
                                              ^
zmalloc.cpp: In function ‘void* ztryrealloc_usable(void*, size_t, size_t*)’:
zmalloc.cpp:246:12: error: invalid conversion from ‘void* (*)(void*, size_t) throw () {aka void* (*)(void*, long unsigned int) throw ()}’ to ‘void*’ [-fpermissive]
     newptr = realloc(realptr,size+PREFIX_SIZE);
```

**To reproduce**

Tested with both ubuntu 16.04 and 18.04. Ran the following commands.
```
git clone git@github.com:EQ-Alpha/KeyDB.git
cd KeyDB
git checkout -b release6 5169f97
make MALLOC=memkind
```

**Expected behavior**

Build success.

**Comment by chibisuke** [comment]
Confirmed. Same problem here.

**Comment by acsfer** [comment]
Legacy FLASH is not supported anymore, and seem not compatible with 6.x codebase.
See also https://community.keydb.dev/t/enabling-legacy-flash/109/2

**Comment by chibisuke** [comment]
Its more like the whole project is dead, as far as responses to bugreports ect. go.

**Comment by benschermel** [comment]
We apologize for some of the delayed responses, please bear with us. We will be ramping up support for open source bugs in the coming months and are currently working on a big open source release scheduled early Q2

**Comment by JohnSully** [comment]
I checked in change d132b27cd which fixes the build breaks with memkind FLASH support.  To be honest this type of FLASH support was a bit of a failed experiment, the kernel doesn't do a good job of choosing which to page and which not to and so you get hit with performance walls once you use too much memory.  We try to help it by differentiating pageable vs non-pageable but the granularity is quite low and it affects every malloc call so it really makes merging new Redis versions difficult.

The criticism of not responding to bugs in a reasonable period is pretty valid.  Most of my time was spent on Pro the last while, and so we're working on open sourcing more of that code as I think we got the balance wrong.  I'm not particularly good at responding to messages, as a result you'll see Ben do more of the initial triage and comments.
